.rubocop_todo.yml

```diff
@@ -83,6 +83,7 @@ Rails/UniqueValidationWithoutIndex:
     - 'app/models/comment.rb'
     - 'app/models/follow.rb'
     - 'app/models/notification.rb'
+    - 'app/models/broadcast.rb'
```

Comment:
Not impacting @juliannatetreault's work here at all, but should we consider removing this `Rails/UniqueValidationWithoutIndex` rule if we're ignoring it in so many files? Is it actually enforcing anything?

Reply:
@vaidehijoshi I'm slowly removing all of those in https://github.com/thepracticaldev/dev.to/pulls?q=is%3Apr+author%3Arhymes+%22unique+indexes%22

I'll remove the rule when we're done, it's a pretty recent one :)